database/entity/0044-adapt_users_table_for_gradidoid/UserContact.ts

```diff
@@ -0,0 +1,40 @@
+import { BaseEntity, Entity, PrimaryGeneratedColumn, Column, DeleteDateColumn } from 'typeorm'
+
+@Entity('user_contacts', { engine: 'InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci' })
+export class UserContact extends BaseEntity {
+  @PrimaryGeneratedColumn('increment', { unsigned: true })
+  id: number
+
+  @Column({
+    name: 'type',
+    length: 100,
+    nullable: true,
+    default: null,
+    collation: 'utf8mb4_unicode_ci',
+  })
+  type: string
+
+  @Column({ name: 'users_id', type: 'int', unsigned: true, nullable: false })
+  usersId?: number | null
```

Comment:
```suggestion
  @Column({ name: 'user_id', type: 'int', unsigned: true, nullable: false })
  userId: number
```
Why to have contacts without a user